spark-commons/src/test/scala/za/co/absa/spark/commons/implicits/StructTypeImplicitsTest.scala

```diff
@@ -216,4 +218,33 @@ class StructTypeImplicitsTest extends AnyFunSuite with SparkTestBase with JsonTe
     assert(schemaB.diffSchema(schemaA).isEmpty)
   }
 
+
+  test("isOfType should return true if the field type matches the expected type") {
+    val path = "/some/path"
+    val results = structTypeEnhancements.isOfType[StringType](path)
+
+    assert(results == true)
+  }
+
+  test("it should return false if the field type does not match the expected type") {
```

Comment:
_"it"_ is ambiguous here. The tests can theoretically run in parallel, and finish in different order.

This form is used in BDD/BDT (Behavior Driven Development/Testing) testing style used for example is here in [`UrisConnectionStringParserSuite`](https://github.com/AbsaOSS/enceladus/blob/develop/utils/src/test/scala/za/co/absa/enceladus/utils/config/UrisConnectionStringParserSuite.scala) (Enceladus) and needs to use `AnyWordSpec` as base class.

This here is a sequence of test cases fully independent.